core/services/users_service.py

```diff
@@ -14,6 +15,15 @@ def create_user(user):
         raise Exception("Failed to create user")
 
 
+def delete_user(user_email):
+    pk_delete_success = delete_partition(user_email)
```

Comment:
Can we lookup the user first to make sure it exists? Raise an exception if not

Reply:
I could be convinced to do that, but wouldn't it be more idempotent to not raise an exception if the user doesn't exist?

Reply:
I think the ideal approach would be to return back to the user with a 400 HTTP status code, but doing thats annoying, so I always just raise exceptions so its at least not a 200. That way the front end doesn't treat it as a success